### ISSUE TITLE
for nerf++ datas ，its pose ,needn't norm？

### ISSUE THREAD

**Issue by Alex-Zhu1** [FREDZEL2020/jittor-MYC-NeRFs#1]
hello，thanks for  you releasing the code combine tensorf and nerf++. 
i notice your code has a radii about intersect_sphere depth for nerf++， that make me confused because of norm in nerf++. 
Scene normalization: move the average camera center to origin, and put all the camera centers inside the unit sphere.  if so, for cars data, its pose make camera look forward to object . 
Looking forward to your reply， or my emails: 1663591752@qq.com

**Comment by FREDZEL2020** [comment]
I didn't do normalization in my scenes, so I left a hyper parameter to tune the nerf++ sphere size for a new unnormalized scene.

**Comment by Alex-Zhu1** [comment]
yeah, thanks your reply.  a large sphere radii can cover the object, which can get a good sence spliting forward area.  can this solution make a good bg_rgb ? the camera pose is look to object, but nerf++ look outside the sphere. 
i apply nerf++ to ngp, its so bad, about 10-20 iters will make loss nan

**Comment by Alex-Zhu1** [comment]
font{
        line-height: 1.6;
    }
    ul,ol{
        padding-left: 20px;
        list-style-position: inside;
    }


    
    
        yeah, thanks your reply. a large sphere radii can cover the object, which can get a good sence spliting forward area. can this solution make a good bg_rgb ? the camera pose is look to object, but nerf++ look outside the sphere.
        
    i apply nerf++ to ngp, its so bad, about 10-20 iters will make loss nan
    
        
            
        
    
    
     
    
        
                    ***@***.***
        
    
 

    On 11/8/2022 ***@***.***> wrote： 



I didn't do normalization in my scenes, so I left a hyper parameter to tune the nerf++ sphere size for a new unnormalized scene.
